fastlane/lanes/release.rb

```diff
@@ -578,8 +577,8 @@ def create_backmerge_pr
 
       #{e.message}
 
-      If this is not the first time you are running the release task, the backmerge PR for the version `#{version}` might have already been previously created.
-      Please close any previous backmerge PR for `#{version}`, delete the previous merge branch, then run the release task again.
+      If this is not the first time you are running the release task, the backmerge PR for the version `#{current_release_version}` might have already been previously created.
+      Please close any previous backmerge PR for `#{current_release_version}`, delete the previous merge branch, then run the release task again.
```

Comment:
What do you think of using `source_branch` in the message? I don't think it's likely, but one _could_ pass a `source_branch` that is unrelated with `current_release_version` and if a failure were to occur, it would not be clear to the reader where to go and look.

```suggestion
      If this is not the first time you are running the automation that creates a backmerge, the backmerge PR for the `#{source_branch}` branch might have already been previously created.
      Please close any previous backmerge PR for the `#{source_branch}` branch, delete the previous merge branch, then run the release task again.
```

Reply:
Good idea 👍 

> Would the change be needed there, too? 🤔

Yes, I'll create a corresponding PR on iOS as well.